web-registry/src/components/molecules/ProjectMetadata/ProjectMetadata.CFC.tsx

```diff
@@ -26,8 +26,8 @@ const ProjectMetadataCFC: React.FC<CFCMetadataProps> = ({
     return null;
   }
 
-  const startDate = metadata?.['regen:projectStartDate']?.['@value'];
-  const endDate = metadata?.['regen:projectEndDate']?.['@value'];
```

Comment:
i'm a bit confused why this was not working, because i thought that the metadata would come through with a `@value` key for this field (based on [the jsonld compaction](https://github.com/regen-network/regen-web/blob/fix-1366-missing-project-date/web-registry/src/lib/metadata-graph.ts#L9) we were doing here)

i checked the metadata that we have on-chain in redwood for this, and ran it through the compaction we are using, and indeed start date and end date are formatting such that they have an `@value` key.

metadata iri obtained from:
```
regen --node http://redwood-sentry.vitwit.com:26657/ q ecocredit project C45-010
```

can be looked up here:
```
http GET http://api-staging.registry.regen.network/metadata-graph/regen:13toVgdVEAmhukPUM1uS5ecN8FYz1XJc5CR7DSjNWPc1EEUKYcneZQV.rdf
```

@mhagel any thoughts on this? am i missing something?

Reply:
@wgwz @mhagel maybe the json-ld corresponding to the C02 project is not complete for those attributes?
![image](https://user-images.githubusercontent.com/5287737/194843435-311cbb1f-ce11-40af-a9fd-0442f22ff5e1.png)

Reply:
Should we close this PR and fix the json-ld in that case ?

Reply:
TBH, I really don't know what is best to do here. I can take a better look at this if it's helpful. But I might need to make some time with @mhagel to review.

Reply:
@flagrede Kyle and I did some debugging and I think we found the issue. I will continue working on this and PR into your branch...

Reply:
@flagrede Actually I had other things to fix, so I propose we close this PR since [this one](https://github.com/regen-network/regen-web/pull/1444) addresses the main problem.

We were displaying the wrong metadata! My mistake overlooking this earlier.